sormas-backend/src/main/java/de/symeda/sormas/backend/immunization/entity/ImmunizationDirectory.java

```diff
@@ -0,0 +1,50 @@
+package de.symeda.sormas.backend.immunization.entity;
+
+import javax.persistence.Entity;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+import de.symeda.auditlog.api.Audited;
+import de.symeda.sormas.backend.vaccination.FirstVaccinationDate;
+import de.symeda.sormas.backend.vaccination.LastVaccinationDate;
+import de.symeda.sormas.backend.vaccination.LastVaccineType;
+
+@Entity(name = "immunizationDirectory")
+@Audited
+@Table(name = "immunization")
+public class ImmunizationDirectory extends BaseImmunization {
```

Comment:
Rename to DirectoryImmunization (because it's not an immunization directory); check whether any of the other classes need to be renamed as well (e.g. DirectoryImmunizationService, etc.)